src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs

```diff
@@ -244,6 +245,117 @@ private static XRefSpec GetXRefInfo(ItemViewModel item, string key)
             {
                 result["fullName.vb"] = item.FullNameForVB;
             }
+            yield return result;
+            // generate overload xref spec.
+            // todo : remove when overload is ready in yaml file.
+            if (item.Type != null)
+            {
+                switch (item.Type.Value)
+                {
+                    case MemberType.Property:
+                    case MemberType.Constructor:
+                    case MemberType.Method:
+                    case MemberType.Operator:
+                        yield return GenerateOverloadXrefSpec(item, key, result);
+                        break;
+                    default:
+                        break;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Work around, remove when overload is ready in yaml file.
+        /// </summary>
+        private static XRefSpec GenerateOverloadXrefSpec(ItemViewModel item, string key, XRefSpec result)
+        {
+            var uidBody = item.Uid;
+            {
+                var index = uidBody.IndexOf('(');
+                if (index != -1)
+                {
+                    uidBody = uidBody.Remove(index);
+                }
+            }
+            uidBody = System.Text.RegularExpressions.Regex.Replace(uidBody, @"``\d+$", string.Empty);
+            var overload = new XRefSpec
+            {
+                Uid = uidBody + "*",
+                Href = key,
+                CommentId = "Overload:" + uidBody,
+            };
+            {
+                var index = item.Name.IndexOfAny(new char[] { '(', '[' });
+                if (index != -1)
+                {
+                    result.Name = item.Name.Remove(index);
+                }
+                else
+                {
+                    result.Name = item.Name;
+                }
+            }
+            if (!string.IsNullOrEmpty(item.NameForCSharp))
+            {
+                var index = item.NameForCSharp.IndexOfAny(new char[] { '(', '[' });
```

Comment:
We will remove this code later.

---
In reply to: [80480966](https://github.com/dotnet/docfx/pull/735#discussion_r80480966) [](ancestors = 80480966)